internal/cmd/context/auth_test.go

```diff
@@ -67,6 +70,15 @@ func TestSetAuth(t *testing.T) {
 				BasicClientSecret: "secret",
 			},
 		},
+		"config with both type of user set": {
+			authName:   "cretendialTest",
+			configPath: copyFile(t, filepath.Join(testdata, "auth.yaml")),
+			options: &clioptions.CLIOptions{
+				BasicClientSecret: "secret",
+				JWTJsonPath:       "path",
+			},
+			expectErr: true,
```

Comment:
Instead of only check the error, it could be better to check also the error message